### ISSUE TITLE
Strange artifact during translation

### ISSUE THREAD

**Issue by AdamJel** [ufal/lindat-translation#60]
Translation from English to Czech of the following:

"Until my request is fulfilled, please restrain your activities in relation to SSO account deletion."

Generates this strange result:

"Do doby, než bude můj požadavek splněn, omezte prosím svou činnost v souvislosti s výmazem SSO účtu. Účet SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO SSO účet
"

**Comment by varisd** [comment]
Indeed. It is interesting, that this does not happen when the final fullstop is removed:

In: "Until my request is fulfilled, please restrain your activities in relation to SSO account deletion"
Out: "Dokud nebude můj požadavek splněn, omezte prosím svou činnost v souvislosti s výmazem SSO účtu"
